.github/workflows/docker.yml

```diff
@@ -23,12 +29,91 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
+    - name: Log in to the GH Container registry
+      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}    
+   
     - name: Set up environment
       run: |
         export GOPATH=$(go env GOPATH)
         echo "GOPATH=$GOPATH" >> $GITHUB_ENV
         echo "$GOPATH/bin" >> $GITHUB_PATH
 
-    - name: Build images
-      run: |
-        make -C dist/images all
+    - name: build ovnkube-binaries copy to context 
+      run: | 
+        pushd go-controller 
+         make 
+        popd 
+
+        pushd dist/images
+          cp -r ../../go-controller/_output/go/bin/* .
+        popd 
+    
+    - name: generate git-info to write to image
+      run: | 
+        BRANCH=$(git rev-parse --short "$GITHUB_SHA")
+        COMMIT=${GITHUB_REF#refs/heads/}
```

Comment:
I don't understand whats going on here. $COMMIT is supposed to reference a commit ? GITHUB_REF can be either pointing to a tag or a branch but not a commit. I guess you mean tag/branch points to a commit? I think having the commit sha is better reference no?

GITHUB_REF env var is described as:
"The branch or tag ref that triggered the workflow run. For branches this is the format refs/heads/<branch_name>, and for tags it is refs/tags/<tag_name>. This variable is only set if a branch or tag is available for the event type. For example, refs/heads/feature-branch-1." [1]
[1] https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
Good catch, this is a mistake it was supposed to be 
`shell git rev-parse  HEAD` to get the commit hash